### PR TITLE
PR for WI #31

### PR DESCRIPTION
**Pull request by palako**
Completing work for work item #31

### CODE REVIEW

File: calculator2.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Calculator2:
+    def add(self, a, b):
```
Comment (palako) [inline calculator2.py:2]: Method names should be prefixed with 'f_'. For example, 'add' should be 'f_add'.

File: calculator2.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+        return a + b
+
+    def subtract(self, a, b):
```
Comment (palako) [inline calculator2.py:5]: Method names should be prefixed with 'f_'. For example, 'subtract' should be 'f_subtract'.

File: calculator2.py (hunk 1 above)
Comment (palako) [inline calculator2.py:2]: Variables should be named using Greek letters. For example, 'a' and 'b' could be 'alpha' and 'beta'.

File: calculator2.py (hunk 2 above)
Comment (palako) [inline calculator2.py:5]: Variables should be named using Greek letters. For example, 'a' and 'b' could be 'alpha' and 'beta'.

File: calculator2.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class Calculator2:
```
Comment (palako) [inline calculator2.py:1]: webhook test

Comment (palako) [reply calculator2.py:1]: @palako mention test in comment

Comment (palako) [reply calculator2.py:1]: testing start a review

File: calculator2.py (hunk 1 above)
Comment (palako) [reply calculator2.py:2]: More comments on the same review

Comment (palako) [reply calculator2.py:2]: another comment

Comment (palako) [reply calculator2.py:2]: one more comment

Comment (palako) [reply calculator2.py:2]: comment

Comment (palako) [reply calculator2.py:2]: comment

File: calculator2.py (hunk 2 above)
Comment (palako) [reply calculator2.py:5]: comment

Comment (palako) [reply calculator2.py:5]: another comment

Comment (palako) [reply calculator2.py:5]: one more

Comment (palako) [reply calculator2.py:5]: more

Comment (palako) [reply calculator2.py:5]: another one

Comment (palako) [reply calculator2.py:5]: more tests

Comment (palako) [reply calculator2.py:5]: more

Comment (palako) [reply calculator2.py:5]: just another one

Comment (palako) [reply calculator2.py:5]: again

File: calculator2.py (hunk 1 above)
Comment (neo-sagittal) [inline calculator2.py:2]: Function names should be prefixed with 'f_'. Please rename 'add' to 'f_add' and 'subtract' to 'f_subtract'.

Comment (neo-sagittal) [inline calculator2.py:2]: Variables should be named using Greek letters. Please rename 'a' and 'b' accordingly.

File: calculator2.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+    def add(self, a, b):
+        return a + b
+
```
Comment (neo-sagittal) [inline calculator2.py:4]: Variables should be named using Greek letters. Please rename 'a' and 'b' accordingly.

**Pull request closed without merging**